src/app/app.menus.ts

```diff
@@ -47,3 +47,11 @@ export const isProjectEditorPredicate = (
  */
 export const isAdminPredicate = (user: User): boolean =>
   isLoggedInPredicate(user) && user.isAdmin;
+
+/**
+ * Returns true only if the user is an admin and can see work in progress items
+ * This predicate differs from the `isAdminPredicate` for semantic reasons
+ * 
+ * @param user User session data. This will be used to check if the user is an admin
+ */
+export const isWorkInProgressPredicate = isAdminPredicate;
```

Comment:
well if it is meant to mirror the WIP component, then it should be something like `env.isDev || isAdminPredicate()`

https://github.com/QutEcoacoustics/workbench-client/blob/152d6dba544751639e61dce4c056e9743cbad958/src/app/components/shared/wip/wip.component.ts#L63-L66